PHPCI/Model/Build.php

```diff
@@ -217,4 +217,28 @@ public function reportError(Builder $builder, $file, $line, $message)
     {
         return array($builder, $file, $line, $message);
     }
+
+    /**
+     * Return the path to run this build into.
+     *
+     * @return string
+     */
+    public function getBuildPath()
+    {
+        return PHPCI_BUILD_ROOT_DIR . $this->getId();
+    }
+
+    /**
+     * Removes the build directory.
+     */
+    public function removeBuildDirectory()
+    {
+        $buildPath = $this->getBuildPath();
+
+        if (!is_dir($buildPath)) {
+            return;
+        }
+
+        exec(sprintf(IS_WIN ? 'rmdir /S /Q "%s"' : 'rm -Rf "%s"', $buildPath));
```

Comment:
Haven't tested it. However, both shell_exec and exec pass the command to cmd.exe on Windows, the only difference being the output handling.